mantidproject/mantid
# Support nonorthogonal axes on > 3D workspaces.

**Description of work.**



**To test:**

See script in the related issue for generating a workspace to test this functionality.
* Try enabling nonorthogonal axes mode
* Ensure that when one of the dimensions selected is not Q then the nonorthogonal button is disabled and then switching back enables the button again.
* When nonorthogonal mode becomes available then the peaks/line viewer should become disabled.

Fixes #28698 

*This does not require release notes* because **nonorthogonal axes are new in this version**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
